joomla-extensions/jedchecker
# Fix regular expressions ("." character should be escaped)

### Summary of Changes
Fix regular expressions ("." character should be escaped)

### Testing Instructions
Add a file with (for example) `.nophp` extension into an extension package, and check it.

### Actual result BEFORE applying this Pull Request
Filename ending with `php` (not `.php`!) are processed as PHP files. The same is true for `xml` file extension.

### Expected result AFTER applying this Pull Request
File extensions are properly processed.

### Documentation Changes Required
None
